### ISSUE TITLE
now extensions don't work rip raga , ovh

### ISSUE THREAD

**Issue by ghost** [davidmann4/ogar-feeder-bot#554]
Please do not ask for proxies
Please do not talk about cheating on public servers.

**Comment by Nick8fr32** [comment]
lol why??

**Comment by ghost** [comment]
when use extension it don't let u to join a party . u can still use tamper monkey

**Comment by LostCodesy** [comment]
Yes, now fixed all extension in agar.io, not use a extension in agar.io for IP ban have in agar.io for bot users.

**Comment by anonymousxxx02** [comment]
I saw some ovh bots before 2hour...are they really patched????

**Comment by anonymousxxx02** [comment]
I just visit ovh site the bots are down

**Comment by Nick8fr32** [comment]
@LostCodesy does ur fix work still??

**Comment by LostCodesy** [comment]
@Nick8fr32 yea my bots fixed now

**Comment by LostCodesy** [comment]
New agar.io patch is too too hard

**Comment by Nick8fr32** [comment]
@LostCodesy can u send?

**Comment by problemsofworld** [comment]
now ur's works like before . screen shot ?
